### PR TITLE
feat: [PIE-8029]: run all icons,images through imageoptim to reduce size, use UICore latest minfied icons version

### PR DESCRIPTION
**Pull request by udayvunnam**
### Summary

Continuation PR for https://github.com/harness/uicore/pull/971

- png format image reduced by about ~50%
- SVG sizes also reduced significantly
- Need to [update snapshots](https://app.harness.io/ng/#/account/vpCkHKsDSxK9_KYfjCTMKA/ci/orgs/default/projects/NextGenUI/pipelines/Jest/executions/qxwXj4XiSpuJhjcV1QNu6Q/tests) `-u` :( that includes icons for uicore/icons upgrade 
 
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by udayvunnam** [comment]
retrigger titlecheck

**Review by chetanvekariya-harness — DISMISSED** [review]

**Review by chetanvekariya-harness — APPROVED** [review]

**Review by SahilKr24 — APPROVED** [review]

**Review by abhinavrastogi-harness — APPROVED** [review]

**Review by bhavya-sinha — APPROVED** [review]

**Comment by udayvunnam** [comment]
retrigger cypress-cd

**Comment by udayvunnam** [comment]
retrigger cypress-pipeline

File: package.json [hunk 1]
```diff
@@ -58,7 +58,7 @@
     "@emotion/styled": "^10.0.27",
     "@harness/design-system": "1.0.0",
     "@harness/help-panel": "1.14.0",
-    "@harness/icons": "^1.103.1",
+    "@harness/icons": "^1.104.1",
```
Comment (mayankVermaHarness) [inline package.json:61]: Nit: Should the version here be `1.104.0` instead of `1.104.1` ?

**Pull request merged**